pandas/tests/io/parser/test_python_parser_only.py

```diff
@@ -66,6 +66,23 @@ def test_sniff_delimiter(python_parser_only, kwargs):
     tm.assert_frame_equal(result, expected)
 
 
+def test_sniff_delimiter_comment(python_parser_only):
```

Comment:
can you move this near other comment tests. & add the issue number as a comment

Reply:
I moved the test to the end of the file, is this what you meant? Or would you rather have me put it in tests/io/parser/test_comment.py?

Reply:
`test_comment.py`

Reply:
Should I move the full test or simply add `dict(sep=None)` as parameter in `test_line_comment` and then skip the c-enginge tests like it's done for the lineterminator test?

Reply:
Move the full test as is and change the fixture from `python_parser_only` to `all_parsers`.

Reply:
This issue affects only `PythonParser`, `sep=None` does not work with the c engine.

Reply:
Oh, I see...

Actually, never mind then.  Per the directive in this file, this test is purely for Python parser-only tests, with the goal of migrating them out of here at later date.

Let's keep the test here in fact.